subhub/cfg.py

```diff
@@ -336,32 +336,42 @@ def PAYMENT_API_KEY(self):
         return self("PAYMENT_API_KEY", "fake_payment_api_key")
 
     @property
-    def SUPPORT_API_KEY(self):
+    def TOPIC_ARN(self):
         """
-        support api key
+        topic arn for sns
+        :return:
         """
-        return self("SUPPORT_API_KEY", "fake_support_api_key")
+        return self("TOPIC_ARN", "fake_arn_key")
```

Comment:
this doesnt follow the pattern laid out before

`SUPPORT_API_KEY` -> `fake_support_api_key`
so it would be
`TOPIC_ARN` -> `fake_topic_arn`

Reply:
Changed to TOPIC_ARN_KEY -> fake_topic_arn_key